tests/legacy-cli/e2e/tests/misc/completion-prompt.ts

```diff
@@ -368,6 +376,54 @@ source <(ng completion script)
       );
     }
   });
+
+  // Prompts when a global CLI install is present on the system.
+  await mockHome(async (home) => {
+    const bashrc = path.join(home, '.bashrc');
+    await fs.writeFile(bashrc, `# Other content...`);
+
+    await execAndWaitForOutputToMatch('ng', ['version'], AUTOCOMPLETION_PROMPT, {
+      ...DEFAULT_ENV,
+      SHELL: '/bin/bash',
+      HOME: home,
+    });
+  });
+
+  // Does *not* prompt when a global CLI install is missing from the system.
+  await mockHome(async (home) => {
+    try {
+      // Temporarily uninstall the global CLI binary from the system.
```

Comment:
I think we are move away from the global installations altogether. At least based on one of Bazel PRs I've seen recently.

//cc @clydin might have more context.

Reply:
Yeah, I was thinking about that too. I'm hoping we're moving away from "always use the global CLI for tests" and doing something a little more local.

Unfortunately for this test, the behavior it's testing is reliant on finding a global CLI on the system, and I don't really see a way around that. Within a Bazel sandbox, we could try to put a CLI in particular place and load it into the `$PATH`, but I'm not sure how involved that will be. We could also fake things and just `touch $SOME_PATH_DIR/ng`, since we don't actually need to execute it in that situation.

The test now is a bit worse since the global CLI already exists and we have to remove it to test things correctly. If Bazel e2e tests don't rely on invoking the CLI via the global `$PATH`, then I think this test becomes a lot easier to write since we just need to add a fake global CLI rather than remove an existing real global CLI.